etcd/etcdtest/mocketcd.go

```diff
@@ -169,8 +170,16 @@ func PutData(key, data string) error {
 
 // Saves a route in etcd with the specified prefix.
 func PutDataTo(prefix, key, data string) error {
+	return PutDataToTTL("/skippertest", key, data, -1)
+}
+
+// Saves a route with TTL in etcd with the specified prefix.
+func PutDataToTTL(prefix, key, data string, ttl int) error {
 	v := make(url.Values)
 	v.Add("value", data)
+	if ttl >= 0 {
```

Comment:
I think it is better to check that it is strictly greater than zero. This way we avoid "magic" constants

Reply:
Makes sense. I changed this to strictly greater than zero.